harmony/s3_stac_io.py

```diff
@@ -0,0 +1,60 @@
+from urllib.parse import urlparse
+import boto3
+from pystac import STAC_IO
+from harmony import util
+from harmony import aws
+from os import environ
+
+"""
+Read and write to s3 when STAC links start with s3://.
+https://pystac.readthedocs.io/en/0.5/concepts.html#using-stac-io
+"""
+
+
+def read(uri):
```

Comment:
Does it make sense to use the more specific 'url' instead of 'uri' here (I realize this is just a parameter name and not a type). This function expects a url based on the parsing below, not a more general uri.